TROUBLESHOOTING.md

```diff
@@ -9,6 +9,19 @@ The **first thing** to check when model is slow is to generate a metrics report.
 Metrics report is extremely helpful in diagnosing issues. Please try to include it in your bug
 report sent to us if you have it.
 
+## Perform A Auto-Metrics Analysis
+
+We provide ways to automatically analyze the metrics report and provide a summary. Simply run your workload with `PT_XLA_DEBUG=1`. Some example output would be
+
+```
+pt-xla-profiler: CompileTime too frequent: 21 counts during 11 steps
+pt-xla-profiler: TransferFromServerTime too frequent: 11 counts during 11 steps
+pt-xla-profiler: CompileTime too frequent: 23 counts during 12 steps
+pt-xla-profiler: TransferFromServerTime too frequent: 12 counts during 12 steps
```

Comment:
It's probably hard to give any more useful info on the compile time problems (you would have to inspect the graph to see what shapes are changing that cause the recompilations).

But do you think it would be useful to add more actionable warnings about data transfers to CPU?

I guess it depends what the source of the data transfers are. If they're usually due to ops that have to fallback to CPU because there isn't a lowering, then it seems pretty useful to include that in the warning. Are there other common reasons for data transfer that would be harder to give better info on?

Reply:
I think for fallback we also have a warning @jysohn23 can confirm

Reply:
Yeah we also have warning for unlowered ops.

Reply:
Maybe I can put a sample message here as well. @jysohn23 Do you want to also answer the `data transfer` issue from Brian 😄

Reply:
What additional info are we thinking of here aside from differentiating user requesting certain tensor data vs CPU fallback? Currently we do log warnings for unlowered ops causing CPU fallback.

Reply:
I mostly just wanted to make sure I had the split right, that those are the only two sources of data transfer. is "user requesting certain tensor data" basically exclusively from either printing tensors / calls to .to('cpu')?

Reply:
I think those are the most common(if not all) of the source of `TransferFromServer`

Reply:
Yes, that should cover like Jack says.